### ISSUE TITLE
[Form] Pressing escape on a form input cause 'Maximum call stack size exceeded' error

### ISSUE THREAD

**Issue by ibelar** [fomantic/Fomantic-UI#1862]
# Bug Report
The error happens when pressing the 'Esc' Key in any form input.

## Steps to reproduce
1. Create a Form with input;
2. Select an input field and press Esc.

## Actual result
Look at browser console for the error:
```
jquery.min.js:2 Uncaught RangeError: Maximum call stack size exceeded
    at String.replace (<anonymous>)
    at X (jquery.min.js:2)
    at G.get (jquery.min.js:2)
    at Ae (jquery.min.js:2)
    at HTMLInputElement.trigger (jquery.min.js:2)
    at Object.trigger (jquery.min.js:2)
    at HTMLInputElement.handler (jquery.min.js:2)
    at HTMLInputElement.dispatch (jquery.min.js:2)
    at HTMLInputElement.v.handle (jquery.min.js:2)
    at Object.trigger (jquery.min.js:2)
```

## Version
2.8.7 using Chrome or Safari on Mac OS 10.15

**Comment by lubber-de** [comment]
That kind of error was recently happening in dropdown fields and fixed by #1661 

Unfortunately, i cannot reproduce this by your given explanation (form with an input).
Please adjust the following jsfiddle. If if click into the input and press escape, nothing happens
https://jsfiddle.net/lubber/vehc039t/1/

**Comment by ibelar** [comment]
@lubber-de  I have update your fiddle.

This is how we are using it:

```
$('.ui.form').form({inline:true, on:"blur"});
 ```

**Comment by lubber-de** [comment]
Thanks. Can reproduce now and will investigate.

**Comment by lubber-de** [comment]
Fixed by #1863
